examples/vucm/ni-daq/script.py

```diff
@@ -0,0 +1,93 @@
+import asyncio
+import csv
+import functools
+import io
+import os
+import socket
+
+import enapter
+
+def parse_bytes(bytes):
+    return bytes.decode("utf-8")

```

Comment:
This function can be replaced with `mybytes.decode()` which is equivalent to `mybytes.decode("utf-8")`.